syntax="proto3";

package serialization;

message FullName{
	string last_name = 1;
	string first_name = 2;
	optional string patronymic = 3;
	}
message Student {
	FullName m_FullName = 1;
	repeated int32 score = 2;
	int32 averageScore = 3;
	}
message StudentsGroup {
	repeated Student Students = 1;
	}